Gemfile

```diff
@@ -8,7 +8,7 @@ gem "winrm-fs"
 gem "activesupport", "~> 4.0"
 
 group :test do
-  gem "rake"
+  gem "rake", "< 12"
```

Comment:
Rake 12 isn't compatible with rubocop 0.28, which is pinned by finstyle 1.4.0.

And the gemspec file says 

> style and complexity libraries are tightly version pinned as newer releases may introduce new and undesireable style choices which would be immediately enforced in CI